modules/ipi-install-configuring-the-install-config-file.adoc

```diff
@@ -97,7 +97,9 @@ for i in $(sudo virsh list | tail -n +3 | grep bootstrap | awk {'print $2'});
 do
   sudo virsh destroy $i;
   sudo virsh undefine $i;
-  sudo virsh vol-delete $i --pool default;
+  sudo virsh vol-delete $i --pool $i;
```

Comment:
```
sudo virsh list | tail -n +3 | grep bootstrap | awk {'print $2'}
```

Only captures the name of the VM with bootstrap in it. $i would never be the pool name. The pool name (by default) is called default.

Reply:
Thank you for review. When I tested with 4.6 in my environment, the volume `$i.ign` is in `default` pool but the volume `$i` is in the pool named `$i`.
In 4.5 and earlier both `$i.ign` and `$i` are in `default` pool. It seems the placement of volumes is changed in 4.6? 

This is my log:

```
[kni@provisioner ~]$ sudo virsh list
 Id   Name                  State
-------------------------------------
 1    ocp-dr8v2-bootstrap   running

[kni@provisioner ~]$ sudo virsh pool-list
 Name                  State    Autostart
-------------------------------------------
 default               active   yes
 ocp-dr8v2-bootstrap   active   yes

[kni@provisioner ~]$ sudo virsh vol-list default
 Name                      Path
----------------------------------------------------------------------------
 ocp-dr8v2-bootstrap.ign   /var/lib/libvirt/images/ocp-dr8v2-bootstrap.ign

[kni@provisioner ~]$ sudo virsh vol-list ocp-dr8v2-bootstrap
 Name                  Path
-------------------------------------------------------------------------------------------------
 ocp-dr8v2-bootstrap   /var/lib/libvirt/openshift-images/ocp-dr8v2-bootstrap/ocp-dr8v2-bootstrap
```

There fore the commands in this PR are required for me to clean up everything in 4.6.

```
[kni@provisioner ~]$ for i in $(sudo virsh list | tail -n +3 | grep bootstrap | awk {'print $2'});
> do
>   sudo virsh destroy $i;
>   sudo virsh undefine $i;
>   sudo virsh vol-delete $i --pool $i;
>   sudo virsh vol-delete $i.ign --pool default;
>   sudo virsh pool-destroy $i;
>   sudo virsh pool-undefine $i;
> done
Domain ocp-dr8v2-bootstrap destroyed

Domain ocp-dr8v2-bootstrap has been undefined

Vol ocp-dr8v2-bootstrap deleted

Vol ocp-dr8v2-bootstrap.ign deleted

Pool ocp-dr8v2-bootstrap destroyed

Pool ocp-dr8v2-bootstrap has been undefined
```